### DISCUSSION TITLE
Echo their are some new extensions i saw

### DISCUSSION THREAD

**Discussion by FlamesGamer12** [3kh0/ext-remover#282] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
so i was using you way to unblock every site then 3 days ago i saw an update for chrome so i thought it would be cool to check it out so i restarted the chrome book I tried to unblock everything again but it didn't work i tried several times but i found out what was going on I checked my Extensions and I saw 2 new Extensions they are called Light Speed Filter Agent and Light Speed Alert Agent so if you can get rid of those that would be awesome thank you for your time

**Comment by ghost** [comment]
Downgrade your chromebook.

**Reply by Fallenmoon8080** [reply]
DON'T DOWNGRADE. YOU'LL GET PUNISHED BY YOUR SCHOOL AND THAT STAYS ON YOUR PERMANENT RECORD.

**Reply by FlamesGamer12** [reply]
how you downgrade

**Reply by ehunt34** [reply]
Make a recovery usb. chrome100.dev

**Comment by Aka-but-nice** [comment]
#347 #348

**Reply by Aka-but-nice** [reply]
tinyurl.com/goofspeed then refer to #348
